Scripts/Generator/RoomGenerator.cs

```diff
@@ -128,4 +169,74 @@ public PhysicsBody GenerateWall(Vector3 wallBase, Vector3 size, Vector3 normal)
         wall.Rotation = new Vector3(0, 1, 0) * normal.AngleTo(Vector3.Forward);
         return wall;
     }
+
+    public void GenerateFloor()
+    {
+        if (floor == null)
```

Comment:
What's this?

Reply:
Somehow even though the floor is initialised in the _Ready, I think Godot tries to set the variables in the Export, which causes the Generate stuff to be called even before _Ready

Reply:
So floor could be null when this is called

Reply:
Ok